docs/config.md

```diff
@@ -0,0 +1,41 @@
+# Configuring gitlint-bot
+
+gitlint-bot lets you control the rules used to analyze commit messages via a configuration file.
+
+## Configuration File
+
+gitlint-bot expects configuration files to be written in YAML with a filename of `.gitlintrc`.
```

Comment:
We should also accept with extension (i.e. `.gitlintrc.yaml`)

Reply:
Should we also accept `.gitlintrc.yml`?

Reply:
Sure, we can support both `.yaml` and `.yml`, and also without extension.